CHANGES.md

```diff
@@ -1,6 +1,440 @@
+* added a new `@gpu.assertElligible` attribute for compile-time GPU checks  
+  (see https://chapel-lang.org/docs/2.2/modules/standard/GPU.html#GPU.@gpu.assertEligible)
+* added a new `@gpu.itersPerThread` attribute to control blocking iterations  
+  (see https://chapel-lang.org/docs/2.2/modules/standard/GPU.html#GPU.@gpu.itersPerThread)
+* added support for GPU attributes to compile without the GPU locale model
+* added a warning when `@assertOnGpu` is used without the GPU locale model
+* added support for `ref` intents on scalars in GPU-eligible loops
+* extended co-locale support to divide GPUs between co-locales
+* extended `make check` to test a GPU-oriented example when GPUs are enabled
+* improved the error message when a vendor's SDK can't be found
+
+Tool Improvements
+-----------------
+* added an `--only` flag to `printchplenv` to focus on a specific variable  
+  (e.g., `$CHPL_HOME/util/printchplenv --only CHPL_TARGET_COMPILER`)
+* made several changes and improvements to the `chplcheck` linter  
+  (see https://chapel-lang.org/docs/2.2/tools/chplcheck/chplcheck.html)
+  - added a new rule for a `complex` literal whose `imag` precedes its `real`
+  - allowed the ability to ignore incorrect indentation within a given scope
+  - added support for a `@fixit` decorator to the Python API
+  - renamed the `RedundantParentheses` rule to `ControlFlowParentheses`
+* added the ability to document interfaces with `chpldoc`
+* updated the script for anonymizing unstable warnings  
+  (see https://chapel-lang.org/docs/2.2/tools/unstableWarningAnonymizer/unstableWarningAnonymizer.html)
+
+Documentation Improvements
+--------------------------
+* linked the module index from the documentation landing page  
+  (see https://chapel-lang.org/docs/2.2/index.html#indexes)
+
+Documentation Improvements for Tools
+------------------------------------
+* added a new 'Editor Support' page  
+  (see https://chapel-lang.org/docs/2.2/usingchapel/editor-support.html)
+* improved the debugging best practices documentation for macOS  
+  (see https://chapel-lang.org/docs/2.2/usingchapel/debugging.html#best-known-configuration)
+* added a section discussing when to use `mason` to the Mason documentation  
+  (see https://chapel-lang.org/docs/2.2/tools/mason/mason.html#when-to-leverage-mason-for-chapel-builds)
+
+Language Specification Improvements
+-----------------------------------
+* documented the process of finding module files through `use` and `import`  
+  (see https://chapel-lang.org/docs/2.2/language/spec/modules.html#finding-toplevel-module-files)
+* added a section describing the implicit `this.` in methods  
+  (see https://chapel-lang.org/docs/2.2/language/spec/methods.html#implicit-this-in-methods)
+* clarified that terminating null bytes are optional on string/bytes buffers  
+  (see https://chapel-lang.org/docs/2.2/language/spec/strings.html  
+   and https://chapel-lang.org/docs/2.2/language/spec/bytes.html)
```

Comment:
These link to the module pages & it's not obvious to me where the change was here. Not necessarily a show-stopper though.

Reply:
I noted this yesterday, but was too lazy to change it.  Since the same thing gave you pause, I did today—thanks!